config.go

```
@@ -102,6 +103,22 @@ func parseFile(filePath string) (*Config, error) {
 		return nil, err
 	}
 
+    // FallBack AAAA Prefixes Validation
+    if len(cfg.FallBacks) > 0 {
+		validPrefixes := []string{}
+		for _, prefix := range cfg.FallBacks {
+			_, _, err := net.ParseCIDR(prefix)
+			if err == nil {
```

So if we didn’t parse the line - we just silently dropped it and that’s it? Without any kind of error message?